x-pack/plugins/cases/docs/openapi/components/schemas/user_activity_response_properties.yaml

```diff
@@ -0,0 +1,74 @@
+type: object
+properties:
+  action:
+    $ref: 'actions.yaml'
+  action_id:
+    type: string
+    example: 22fd3e30-03b1-11ed-920c-974bfa104448
+  case_id:
+    type: string
+    example: 22df07d0-03b1-11ed-920c-974bfa104448
+  comment_id:
+    type: string
+    example: 578608d0-03b1-11ed-920c-974bfa104448
+  created_at:
+    type: string
+    format: date-time
+    example: 2022-05-13T09:16:17.416Z
+  created_by:
+    type: object
+    properties:
+      email:
+        type: string
+        example: null
+      full_name:
+        type: string
+        example: null
+      username: 
+        type: string
+        example: elastic
+  owner:
+    $ref: 'owners.yaml'
+  payload:
+    type: object
+    properties:
+      comment: 
+        type: object
+        properties:
+          comment:
+            type: string
+            example: A new comment.
+          owner:
+            $ref: 'owners.yaml'
+          type: 
+            type: string
+            example: user
+      connector: 
+        type: object
+        properties:
+          $ref: 'connector_properties.yaml'
+      description:
+        type: string
+      owner:
+        $ref: 'owners.yaml'
+      settings:
+        type: object
+        properties:
+          syncAlerts:
+            type: boolean
+            example: true
+      severity:
+        $ref: 'severity_property.yaml'
+      status: 
+        $ref: 'status.yaml'
+      tags: 
+        type: array
+        items:
+          type: string
+        example: ["tag-1"]
+      title:
+        type: string
+    additionalProperties: true
```

Comment:
>    additionalProperties: true

Note that I added this property since I wasn't certain that I captured every payload combination for each action and type. If there's a definitive list somewhere, please let me know and I can remove this property.

Reply:
Hey Lisa! Yes, some are missing. Depending on the user action the payload changes. Here are all supported user action payloads`https://github.com/elastic/kibana/tree/main/x-pack/plugins/cases/common/api/cases/user_actions`. Some of the maybe difficult to understand. Let me know if you need help with that.

Reply:
For the comments payload, I would only list the comment (as you already have) and the alerts. Let's hold on for the moment for the other types. We just added some in 8.4 and are considered an implementation detail.

Reply:
Thanks!  I've added `payload_<type>.yaml` files for each of the action types. If one of those contains the type of implementation details that you want to omit, just let me know!